code/util/gather.c

```diff
@@ -106,6 +106,10 @@ void allocbank(void)
 	++allocated;
 }
 
+int comp(const void *a, const void *b) {
+    return indexes[*(int *)a] - indexes[*(int *)b];
+}
+
```

Comment:
NO

Reply:
This was done because gcc in Mac says:
```
gather.c:274:41: error: function definition is not allowed here
        int comp(const void *a, const void *b) {
```

Reply:
I have an easy fix: wrap this in `#ifdef __STDC__` and then wrap the other in `#ifndef __STDC__`.

Reply:
Good catch! Didn't know about that preprocessor directive. Added to avoid moving code.